# Human methylstion atlas
A research using FACS and Whole Genome Bisulfite Sequencing generate a DNA methylation atlas of 39 main human cell types. Surprisingly, what we should keep an eye on are not methylated regions, but are those unmethylated ones. The "differentially-unmethylated" regions contain important regulatory elements in cell development and bound by cell-type-specific master transcription factors. Those unmethylated DNA fragments could even be used as cell-type-specific biomarkers in liquid biopsies (cfDNA). Thanks to Netanel et.al for this great research.

Here are some highlight for this human DNA methylation atlas.

- From 137 donors, 205 samples covering 77 cell types.
- > 90% purity for each cell population.
- 205 methylomes.
- Deep sequencing (average coverage = 30X, min. 6.62X).
- The entire human genome was first segmented as 7104162 blocks.
- 2783421 methylation blocks were detected (3-8 CpG / 544 bp).
- Only 1% of markers are covered by RRBS.
- 99% similarity between biological replicates.
- Total 1246 differentially-methylated DNA loci could be used as markers for total 39 main cell types.
- Differentially-unmethylated regions are mostly enhancers and bound by cell-type-specific master TFs.
- Differentially-hypermethylated regions are CpG islands, polycomb targets, and CTCF binding site.
- 1246 DNA methylation atlas markers could be used as a reference data for deconvolution analysis.
- Cell proportions could be determined with deconvolution analysis by those 1246 DNA methylation atlas markers and serve as a potential parameters of disease severity (e.g. COVID19).

[Credit](https://www.nature.com/articles/s41586-022-05580-6)
